packages/core/admin/admin/src/pages/ProfilePage/tests/index.test.js

```diff
@@ -0,0 +1,1062 @@
+                          </div>
+                          <div
+                            class="c33"
+                          >
+                            <button
+                              aria-disabled="false"
+                              aria-label="Clear the interface language selected"
+                              class="c36"
+                            >
+                              <svg
+                                fill="none"
+                                height="1em"
+                                viewBox="0 0 24 24"
+                                width="1em"
+                                xmlns="http://www.w3.org/2000/svg"
+                              >
+                                <path
+                                  d="M24 2.417L21.583 0 12 9.583 2.417 0 0 2.417 9.583 12 0 21.583 2.417 24 12 14.417 21.583 24 24 21.583 14.417 12 24 2.417z"
+                                  fill="#212134"
+                                />
+                              </svg>
+                            </button>
+                            <button
+                              aria-hidden="true"
+                              class="c37 c36 c38"
+                              tabindex="-1"
+                            >
+                              <svg
+                                fill="none"
+                                height="1em"
+                                viewBox="0 0 14 8"
+                                width="1em"
+                                xmlns="http://www.w3.org/2000/svg"
+                              >
+                                <path
+                                  clip-rule="evenodd"
+                                  d="M14 .889a.86.86 0 01-.26.625L7.615 7.736A.834.834 0 017 8a.834.834 0 01-.615-.264L.26 1.514A.861.861 0 010 .889c0-.24.087-.45.26-.625A.834.834 0 01.875 0h12.25c.237 0 .442.088.615.264a.86.86 0 01.26.625z"
+                                  fill="#32324D"
+                                  fill-rule="evenodd"
+                                />
+                              </svg>
+                            </button>
+                          </div>
+                        </div>
+                      </div>
+                      <p
+                        class="c39"
+                        id="select-1-hint"
+                      >
+                        This will only display your own interface in the chosen language.
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </form>
+      </main>
+    `);
+  });
+
+  it('should display username if it exists', async () => {
+    render(App);
+    await waitFor(() => {
+      expect(screen.getByText('yolo')).toBeInTheDocument();
+    });
+  });
+
+  it("should display firstname/lastname if username doesn't exist", async () => {
```

Comment:
You can remove this one. or simply put `test.todo('should display firstname/lastname when the username is null')`